### ISSUE TITLE
Sweep: extra ffnn-senses.py CLI arguments

### ISSUE THREAD

**Issue by solresol** [solresol/ultratree-neural-baseline#9]
### Details
ffnn-senses.py has these global variables:
EMBEDDING_DIM = 128     # Dimension of the embeddings
CONTEXT_SIZE = 16       # Number of context word senses
HIDDEN_DIM = 256        # Dimension of the hidden layer
BATCH_SIZE = 1024       # Batch size for training
NUM_EPOCHS = 1000       # Maximum number of epochs for training
LEARNING_RATE = 0.001   # Learning rate for optimizer

They should be controllable by cli arguments. (The current values are suitable as defaults to those CLI arguments.)

<details open>
<summary>Checklist</summary>

- [X] Modify `ffnn-senses.py` ✓ https://github.com/solresol/ultratree-neural-baseline/commit/e4587b8941f16129238a5ef1454b710d39d07d27 [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L101-L107)
- [X] Running GitHub Actions for `ffnn-senses.py` ✓  [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L101-L107)
- [X] Modify `ffnn-senses.py` ✓ https://github.com/solresol/ultratree-neural-baseline/commit/55204caa663f57c50045416915bc16ca5a52037c [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L130-L137)
- [X] Running GitHub Actions for `ffnn-senses.py` ✓  [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L130-L137)
- [X] Modify `ffnn-senses.py` ✓ https://github.com/solresol/ultratree-neural-baseline/commit/1be7762d64913b8a76eb2fb00bde48102f4e63c3 [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L140-L141)
- [X] Running GitHub Actions for `ffnn-senses.py` ✓  [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L140-L141)
- [X] Modify `ffnn-senses.py` ✓ https://github.com/solresol/ultratree-neural-baseline/commit/3b05fc36b22129127058019e8c9dfcb48323831b [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L126-L127)
- [X] Running GitHub Actions for `ffnn-senses.py` ✓  [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L126-L127)
- [X] Modify `ffnn-senses.py` ✓ https://github.com/solresol/ultratree-neural-baseline/commit/6d613e148d2108a4eeff8570493d28ecaf7f624b [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L155-L158)
- [X] Running GitHub Actions for `ffnn-senses.py` ✓  [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L155-L158)
- [X] Modify `ffnn-senses.py` ✓ https://github.com/solresol/ultratree-neural-baseline/commit/07e57d87338686eaa9525f618131f421fa957afe [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L197-L201)
- [X] Running GitHub Actions for `ffnn-senses.py` ✓  [Edit](https://github.com/solresol/ultratree-neural-baseline/edit/sweep/extra_ffnnsensespy_cli_arguments/ffnn-senses.py#L197-L201)
</details>
